docs/source/blocks/block-style-wrapper.md

```diff
@@ -218,8 +218,11 @@ Finally, assuming that you select the default value for the {guilabel}`Aspect Ra
 </div>
 ```
 
-The custom CSS property definition will only apply within the div that it's defined.
-As you can see, the custom CSS property applies only within the `div` in which it is defined.
+As you can see, the custom CSS declaration applies only in the `div` where you inject the property.
```

Comment:
@stevepiercy the reason it was mentioned as CSS property definition instead of the custom CSS declaration is due to how this feature is called in CSS.

I would write as 
The CSS property will be applied on the `div` where you inject the property.

This is because this property could affect also its children, you don't know where inside that section someone has a var() call that makes use of it

Reply:
Thanks for the clarification and the assist. Let's merge this!